third_party/Makefile

```
@@ -5,6 +5,8 @@
 
 SB_MICRO_PATH ?= "/usr/local"
 MPI_HOME_PATH ?= "/usr/local/mpi"
```

rename to `MPI_HOME`? and better to [remove quotes](https://stackoverflow.com/a/23332194/7630831)
```suggestion
MPI_HOME ?= /usr/local/mpi
```

---

ok, rename to MPI_HOME and remove all quotes of path in vars